developer_manual/app_development/bootstrap.rst

```diff
@@ -19,8 +19,8 @@ The `Application` class is the main entry point of an app. This class is optiona
 to register any services or run some code for every request.
 
 
-Nextcloud will try to autoload the class from the namespace ``\OCA\<App namespace>\AppInfo\Application``, like e.g.
-``\OCA\Mail\AppInfo\Application``. The file will therefore be located at ``lib/AppInfo``.
+Nextcloud will try to autoload the class from the namespace ``\OCA\<App namespace>\AppInfo\Application``, like 
+``\OCA\MyApp\AppInfo\Application``, where *MyApp* would be the name of your app. The file will therefore have the location ``MyApp/lib/AppInfo/Application.php``. 
```

Comment:
```suggestion
``\OCA\MyApp\AppInfo\Application``, where *MyApp* would be the name of your app. The file will therefore have the location ``myapp/lib/AppInfo/Application.php``. 
```

The appid is lowercase. The namespace itself is also specified in info.xml to allow proper casing. :)

Reply:
ah ok, missed that

Reply:
@PanCakeConnaisseur please fix :)

Reply:
Oh, I didn't know it was up to me. Thought, it would be accepted automatically. Thanks for the hint.